.travis.yml

```diff
@@ -27,15 +18,24 @@ after_failure:
     echo $pod
     kubectl logs --namespace=$BINDER_TEST_NAMESPACE $pod
   done
+
 after_success:
 - codecov
-- |
-  if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
-    docker login -u ${DOCKER_USERNAME} -p "${DOCKER_PASSWORD}"
-    cd helm-chart
-    ./build.py --commit-range ${TRAVIS_COMMIT_RANGE} --push
-    cd ..
-  fi
+
+jobs:
+  include:
+  - stage: deploy
+    branches:
+      only:
+      - master
```

Comment:
That does not seem to work as right now travis tried to decrypt things and failed now waitign for DockerHub username.